vendor/fmt/src/dune

```diff
@@ -0,0 +1,5 @@
+(library
+ (name fmt)
+ (modules fmt)
+ (flags :standard -w -3-6-27-34-50)
```

Comment:
How come we need the flags? We ignore all warnings in the vendored dir anyway.

Reply:
Ah these are copied from the dune file (from the dune-universe port). Will remove